src/builtins/Transformers.jl

```diff
@@ -516,58 +518,50 @@ MLJBase.output_is_multivariate(::Type{<:OneHotEncoder}) = true
 
 ## Imputer
 """
-    fillImputer -   imputes by applying function to non missing values
+    FillImputer -   imputes by applying function to non missing values
                     default for continuous(median), CategoricalArray(most common),
                         Count (rounded median)
 
 
 """
-
```

Comment:
To be consistent, we should say "MultiClass" instead of "CategoricalArray". And what about OrderedFactor? Do you want a separate imputing strategy, or maybe combine with "Count"? Probably simpler to understand if it has it's own strategy.